modules/generator/registry/native_histogram.go

```diff
@@ -180,14 +208,15 @@ func (h *nativeHistogram) collectMetrics(appender storage.Appender, timeMs int64
 			return activeSeries, err
 		}
 
-		// NOTE: Store the encoded histogram here so we can keep track of the exemplars
-		// that have been sent.  The value is updated here, but the pointers remain
-		// the same, and so Reset() call below can be used to clear the exemplars.
+		// NOTE: Store the encoded histogram here so we can keep track of the
+		// exemplars that have been sent.  The value is updated here, but the
+		// pointers remain the same, and so Reset() call below can be used to clear
+		// the exemplars.
 		s.histogram = encodedMetric.GetHistogram()
 
 		// If we are in "both" or "classic" mode, also emit classic histograms.
 		if hasClassicHistograms(h.histogramOverride) {
-			classicSeries, classicErr := h.classicHistograms(appender, lb, timeMs, s)
+			classicSeries, classicErr := h.classicHistograms(appender, s.lb, timeMs, s)
```

Comment:
since s.lb is a member var it could be removed from the function signature?

keeping it as a param might make it easier to test?

Reply:
Good call.  Tests cover this without needing the param.